runtime/src/main/java/com/redhat/ipaas/rest/Users.java

```diff
@@ -37,12 +40,16 @@
     @Inject
     private DataManager dataMgr;
 
+    @Context
```

Comment:
This is just a matter of personal taste, but I prefer this to be a parameter to the method, just feels a bit more readable to me for request scoped context. What do you think?